sormas-rest/src/main/java/de/symeda/sormas/rest/CaseResource.java

```diff
@@ -143,4 +145,21 @@ public Response updateExternalData(List<ExternalDataDto> externalData) {
 		}
 	}
 
+	/**
+	 * This endpoint is used to partially update the CaseData.
+	 * For allowing only a subset of the fields of the caseDataDto to be updated, and ensuring that the system can determine
+	 * which fields are not provided and which are intended to be reset it, the payload needs to be a Json object and not a dto
+	 *
+	 * @param uuid
+	 * @param caseDataDtoJson
+	 *            - a subset of caseDataDto fields, same structure as caseDataDto
+	 * @return - the updated caseDataDto
+	 * @throws Exception
+	 */
+	@POST
+	@Path("/postUpdate/{uuid}")
+	public CaseDataDto postUpdate(@PathParam("uuid") String uuid, JsonNode caseDataDtoJson) throws Exception {
+		return FacadeProvider.getCaseFacade().postUpdate(uuid, caseDataDtoJson);
```

Comment:
@marius2301 We have to make sure that the provided json does not override the uuid of existing entities. Also not the uuid of existing child entities (e.g. symptoms of a case).